atomicapp/cli/main.py

```diff
@@ -53,9 +53,10 @@ def print_app_location(app_path):
 def cli_genanswers(args):
     try:
         argdict = args.__dict__
+        location = argdict['answers']
```

Comment:
If we are to leave it as `-a` then I'd rather not do this here but rather leave it as `answers` and comment in the function in `main.py` what it is being used for.